backend/src/services/v2/smtp/smtp.ts

```diff
@@ -18,20 +18,67 @@ export async function requestReviewForRelease(entity: string, review: ReviewDoc,
     return
   }
 
-  const releaseName = getReleaseName(release)
+  const emailContent = buildEmail(
+    `Release ${release.semver} has been created for model ${release.modelId}`,
+    [
+      { title: 'Model ID', data: release.modelId },
+      { title: 'Your Role', data: review.role.toUpperCase() },
+      { title: 'Semver', data: release.semver },
+      { title: 'Created By', data: release.createdBy },
+    ],
+    [
+      { name: 'Open Release', url: 'TODO' },
+      { name: 'See Reviews', url: 'TODO' },
+    ],
+  )
+
+  const userInfoList = await Promise.all(await authorisation.getUserInformationList(entity))
+  const sendEmailResponses = userInfoList.map(
```

Comment:
I think it'd be valuable to have some cut-off on the number of emails we can send in a single batch.